client/src/main/scala/org/http4s/client/websocket/WSClient.scala

```diff
@@ -130,7 +130,13 @@ private[http4s] trait WSConnection[F[_]] { outer =>
   }
 }
 
-private[http4s] trait WSConnectionHighLevel[F[_]] { outer =>
+trait WSConnectionHighLevel[F[_]] { outer =>
+
+  /** Send a single websocket text frame. The sending side of this connection has to be open. */
+  def send(text: String): F[Unit] = send(WSFrame.Text(text))
+
+  /** Send a single websocket binary frame. The sending side of this connection has to be open. */
+  def send(bytes: ByteVector): F[Unit] = send(WSFrame.Binary(bytes))
```

Comment:
Thoughts on overloads vs `sendText` and `sendBytes`?

Reply:
I tend to avoid overloads these days.

Reply:
Yeah, it's good practice. But I had to try :P